### ISSUE TITLE
Add support for carry weight reduction

### ISSUE THREAD

**Issue by Inmortuae** [LennardF1989/BG3-BagsOfSorting#2]
So i know there is a mod for the bag of holdings which lets you hold more by lowering the weight of each item added and then returning it back to its normal weight after taken out. kinda like the chest of mundane. i was just wondering if that would be possible to add like weight reduction in and for the bags. could be set as an option for those who dont want the hoarding feeling

**Comment by LennardF1989** [comment]
Good idea, I had that somewhere in the back of my mind too :) I'll see if I can get it in with the next update.

EDIT: It's a lot more involved that I thought, so maybe not next update :P

**Comment by Inmortuae** [comment]
I thought it was a bit complex if i can be of any assistance lmk

**Comment by jrieko** [comment]
Would it be possible to have a BOS bag extend the BOH item? I'm now trying to set it as such manually. I.e. after generating the pak, edit the source files
- use the BOH mod's item (7471c9a3-e826-4ff2-b04c-b47c5360e46f) as a `ParentTemplateId`. The [BOH lua script](https://github.com/Auridh/bg3_bagOfHolding/blob/main/mod/Mods/Auridh_BoH/ScriptExtender/Lua/Server/BagOfHolding.lua#L15) seems to condition on the `Stats` attribute, so as long as it inherits that from the BOH parent, I suspect it will match.
- `using "BoH_OBJ_BagOfHolding"` in the Stats Data file
- then re-pak

I'm brand new to modding BG3/Osiris API, so I don't doubt this isn't how it works. I'll reply after trying this.

**Comment by jrieko** [comment]
I've verified this does indeed work. I tested all of BOH's features behave correctly with my modified BOS container. To document what I did:
1. BOS, create bags, pak as usual. I also added 1 BOH item to the POW treasure table, but I doubt that's required for this to work.
1. `Public\GustavDev\RootTemplates\BOS_...lsx`: 
    - `<attribute id="ParentTemplateId" type="FixedString" value="7471c9a3-e826-4ff2-b04c-b47c5360e46f" />`
    - delete the `Stats` attribute, so it inherits from the BOH parent
    - LSLib toolkit, convert this lsx file to update the lsf file
1. `Public\GustavDev\Stats\Generated\Data\BOS_...txt`: 
    - `using "BoH_OBJ_BagOfHolding"`
1. pak it up
1. Add both BOH and BOS to BG3MM. Order BOS after BOH.

**Comment by LennardF1989** [comment]
Good to know! I'll see if I can incorporate this :) I'm pretty sure step 3 is superfluous if you removed the Stats entry. The ParentTemplateId is really just how the item should look like (and which Stats to use). The Stats actually tell how it "reacts". So instead of Step 2, Step 3 should be enough.

**Comment by LennardF1989** [comment]
@jrieko Can you confirm that what I said is correct? Saves me having to test myself since I don't have script extender installed (yet).
